Simplenote/Classes/FileManager+Simplenote.swift

```diff
@@ -13,4 +13,16 @@ extension FileManager {
 
          return url
      }
+
+    /// URL for Simplenote's shared app group directory
+    ///
+    var sharedContainerURL: URL {
+        containerURL(forSecurityApplicationGroupIdentifier: Constants.sharedDirectoryDomain + Constants.groupIdentifier)!
+    }
+}
+
+private struct Constants {
+    static let defaultBundleIdentifier = "com.codality.NationalFlow"
```

Comment:
It should be `com.codality.NotationalFlow` I think

Reply:
haha.  You know I have always wondered why it was national flow.... answer, cause it isn't :duh: notational flow makes so much more sense!

Reply:
LOL